package.json

```diff
@@ -1,7 +1,15 @@
 {
   "name": "root",
-  "npmClient": "npm",
+  "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
+    "build": "npm-run-all prettier:check clean build:mgt-element build:mgt",
```

Comment:
this seems hard to scale when we add more projects, is there reason why lerna run wouldn't work here?

Reply:
It's also not working perfectly. I need to think about this more

Reply:
I've switched over to `lerna run` :)